service/src/test/java/bio/terra/workspace/common/StairwayTestUtils.java

```diff
@@ -53,7 +53,7 @@ public static FlightState blockUntilFlightCompletes(
 
     stairway.submitWithDebugInfo(
         flightId, flightClass, inputParameters, /* shouldQueue= */ false, debugInfo);
-    return pollUntilComplete(flightId, stairway, timeout.dividedBy(20), timeout);
+    return pollUntilComplete(flightId, stairway, Duration.ofSeconds(1), timeout);
```

Comment:
This should reduce overall test time. The timeout is currently hardcoded to 75m so this polling interval was set to 3.75m meaning any test that calls this waits a minimum of that time. This is a simple db lookup, doing it once per second should not be a problem.